<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrivilegeWidget</class>
 <widget class="QWidget" name="PrivilegeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>34</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PrivilegeWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QCheckBox" name="instrumentCheckBox">
     <property name="text">
      <string>仪器管理</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="instrumentProductionCheckBox">
     <property name="text">
      <string>仪器生产管理</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="reagentCheckBox">
     <property name="text">
      <string>试剂管理</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="reagentProductionCheckBox">
     <property name="text">
      <string>试剂生产管理</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="hospitalCheckBox">
     <property name="text">
      <string>医院管理</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="accountCheckBox">
     <property name="text">
      <string>账号管理</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
